src/component/Dragon/DragonList.js

```diff
@@ -0,0 +1,47 @@
+import './dragon.css';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
```

Comment:
- [ ] Same here, Please remove this eslint disablement line and fix linters issues yourself. Linters is a tool to help you write clean and error-free code so please don't disable it and try to fix it's errors.